include/dem/insertion.h

```diff
@@ -65,10 +65,15 @@ class Insertion
    * classes.
    *
    * @param distribution_object_container Contains all distribution for each
-   * type of particle
+   * particle type
+   * @param triangulation Triangulation to access the cells in which the
```

Comment:
We have an extra line after each parameter in general, no?

Reply:
I have just looked, and no.